conda/libmesh/build.sh

```diff
@@ -75,9 +75,9 @@ fi
 source $SRC_DIR/configure_libmesh.sh
 export INSTALL_BINARY="${SRC_DIR}/build-aux/install-sh -C"
 METHODS="opt oprof devel dbg" configure_libmesh --prefix=${PREFIX}/libmesh \
+                                                --with-future-timpi-dir=${PREFIX}/libmesh \
```

Comment:
```suggestion
```
Likewise, move to `configure_libmesh.sh`

Reply:
Withdrawing my comment. I see your intent with "future" TIMPI dir now.

Reply:
I guess I don't follow this completely

Reply:
Maybe https://github.com/roystgnr/MetaPhysicL/pull/58 can help explain. We have to make sure that METAPHYSICL_HAVE_TIMPI is defined in metaphysicl_config.h otherwise parallel metaphysicl code will not be available

Reply:
This PR relies on the ability to send and receive dual numbers